src/App/Link.cpp

```diff
@@ -325,11 +325,8 @@ void LinkBaseExtension::setProperty(int idx, Property* prop)
         if (!prop) {
             propName = "<null>";
         }
-        else if (prop->getContainer()) {
-            propName = prop->getName();
-        }
         else {
-            propName = extensionGetPropertyName(prop);
```

Comment:
I don't think this will be the right solution - this effectively takes away ability to affect property names from extensions which may be used by some extensions (or someone may want to use it at some point).

Probably would be better to address that in an extension that overrides this method and causes problems as this seems to come from extension.